azure/k8s/ctfd-values.yaml

```diff
@@ -0,0 +1,162 @@
+# Default values for ctfd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
@bendehaan can you check please if the end-state of the cluster brings up the right values for ctfd using the script:) ?